### ISSUE TITLE
Intent Drafts: Suggest getting a review

### ISSUE THREAD

**Issue by inexorabletash** [GoogleChrome/chromium-dashboard#1104]
Request: Encourage having Intent draft emails reviewed by others before sending

**The problem**

It's a frequent occurrence that Intents are sent out with "obvious" mistakes - e.g. questions incorrectly or insufficiently answered, poor wording, etc. The consequence of this ranges from round-trips with API owners asking for more details, to creating unnecessary tension with other browser engine implementers (e.g. by misrepresenting positions). 

On my team we have a process that requires that Intents should at least be reviewed by a spec mentor prior to sending. Unfortunately, the Intent tooling in chromestatus makes the output look "final" enough that people skip getting a review.

**Possible solutions**

Per https://github.com/GoogleChrome/chromium-dashboard/issues/823#issuecomment-736865554 you could imagine that drafts can be saved and collaborated on by reviewers. That's a big project.

The current tool could benefit from a range of minor things, e.g.:

* Informational: A _"TIP: Get your draft reviewed by a spec mentor before sending!"_ box on the side.
* Actionable: A no-op checkbox outside the form reading e.g. "I had my draft reviewed by a spec mentor"

Basically... anything to make the intents that get sent out higher quality, to avoid unnecessary confusion/churn.

**Comment by jpmedley** [comment]
For the informational, how about something like this:

![image](https://user-images.githubusercontent.com/8204171/101184122-306b3200-3605-11eb-9be8-db66a6569b51.png)

**Comment by scheib** [comment]
LGTM

**Comment by jpmedley** [comment]
@inexorabletash What do you think?

**Comment by jpmedley** [comment]
Josh,

Does this solution look good to you?

**Comment by inexorabletash** [comment]
Week, sorry, thanks for the re-ping. Don't be shy about doing that, I get/ignore a lot of github pings.

Yeah, this LGTM.

**Comment by jpmedley** [comment]
@jrobbins Before I update this, are we emailing from within the tool yet? I thought we were.
